### ISSUE TITLE
Борг может вставить свой анализатор атмосферы в камеру наблюдения

### ISSUE THREAD

**Issue by T6751** [TauCetiStation/TauCetiClassic#6147]
#### Подробное описание проблемы
Борг может вставить свой анализатор атмосферы в камеру наблюдения
#### Что должно было произойти
Не может, так как является модулем борга
#### Что произошло на самом деле
Анализатор спокойно вставился в камеру, тем самым оставив борга без анализатора
#### Как повторить
Будучи боргом, открутить камеру, выбрать анализатор атмосферы, кликнуть по камере
#### Дополнительная информация:

**Comment by Ambassadoing** [comment]
Так, стоять. Как это в камеры можно вставлять анализатор атмосферы? Что это даёт?

**Comment by AirBlack** [comment]
дает икс рей камере как при взломе мальфом

**Comment by AzzyDreemurr13** [comment]
Cпустя 3 года фиксим недоразумение

**Comment by TgT4** [comment]
фиксим жрущие инструменты автолаты, колбы хим машины, сигналлеры, анализаторы и всё прочее

**Comment by TauKitty** [comment]
Эта проблема была упомянута в **Tau Ceti Station Forum**. Там могут быть соответствующие подробности:

https://forum.taucetistation.org/t/banhammer-nauchnyj-borg/43152/3
